### PR TITLE
Added rooftops .svg and phone .svg

### PR DESCRIPTION
**Pull request by TheGemDev**
Added rooftops .svg and phone .svg to the project folder

### CODE REVIEW

File: src/assets/rooftop/svg/vecteezy_here-is-a-very-useful-phone-charger-vector-set-hope-you-can-use-these-in-your-project-enjoy_105508.svg [hunk 1]
```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (arthuro555) [inline src/assets/rooftop/svg/vecteezy_here-is-a-very-useful-phone-charger-vector-set-hope-you-can-use-these-in-your-project-enjoy_105508.svg:1]: That's a very long file name, would it be possible to shorten it?

File: src/assets/rooftop/svg/ROOFTOPS_1_046 [Converted].svg [hunk 2]
```diff
@@ -0,0 +1,627 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (arthuro555) [inline src/assets/rooftop/svg/ROOFTOPS_1_046 [Converted].svg:1]: Maybe remove the [CONVERTED] from the title

**Review by arthuro555 — APPROVED** [review]
This looks fine, but I'd recommend you use more descriptive names to the svgs

File: src/assets/rooftop/svg/vecteezy_here-is-a-very-useful-phone-charger-vector-set-hope-you-can-use-these-in-your-project-enjoy_105508.svg (hunk 1 above)
Comment (TheGemDev) [reply src/assets/rooftop/svg/vecteezy_here-is-a-very-useful-phone-charger-vector-set-hope-you-can-use-these-in-your-project-enjoy_105508.svg]: Done

File: src/assets/rooftop/svg/ROOFTOPS_1_046 [Converted].svg (hunk 2 above)
Comment (TheGemDev) [reply src/assets/rooftop/svg/ROOFTOPS_1_046 [Converted].svg]: done :+1:

**Pull request merged**